views/faq.handlebars

```diff
@@ -12,6 +12,7 @@
             <li>Recent setlists: Lineup List will look up the songs that this artist performed at past shows and choose songs from them, resulting in songs more relevant to what the artist performs currently</li>
             <li>Newest tracks: The most recent releases from the artist will be used</li>
         </ul>
+        <p>If you want to change anything later, just re-run the process - it will recreate the Spotify playlist from scratch each time.</p>
```

Comment:
```suggestion
        <p>If you want to change anything later, just re-run the process. If you haven't renamed the Spotify playlist it will create a new one from scratch and overwrite it. If you change the name of the generated playlist, Lineup Lists will generate an all-new one.</p>
```